pkg/rke2/clusterrole_bootstrap.go

```diff
@@ -37,10 +37,11 @@ func clusterRoles() []rbacv1.ClusterRole {
 			ObjectMeta: metav1.ObjectMeta{Name: tunnelControllerName},
 			Rules: []rbacv1.PolicyRule{
 				rbacv1helpers.NewRule("get").Groups(legacyGroup).Resources("nodes").RuleOrDie(),
-				rbacv1helpers.NewRule("list", "watch").Groups(legacyGroup).Resources("namespaces").RuleOrDie(),
+				rbacv1helpers.NewRule("list", "watch", "get").Groups(legacyGroup).Resources("namespaces").RuleOrDie(),
 				rbacv1helpers.NewRule("list", "watch").Groups(networkingGroup).Resources("networkpolicies").RuleOrDie(),
 				rbacv1helpers.NewRule("list", "watch", "get").Groups(legacyGroup).Resources("endpoints", "pods").RuleOrDie(),
-				rbacv1helpers.NewRule("list", "get").Groups(legacyGroup).Resources("secrets").RuleOrDie(),
+				rbacv1helpers.NewRule("list", "get").Groups(legacyGroup).Resources("secrets", "serviceaccounts").RuleOrDie(),
```

Comment:
I think we can drop get access to `"serviceaccounts"` since we didn't end up needing that?

Reply:
Yes, I am also removing the `get` I added for namespaces as that was part of testing